src/main/java/command/Commands.java

```diff
@@ -0,0 +1,351 @@
+package command;
+
+
+import task.ListOfTask;
+import dukeExceptions.DukeException;
+
+import java.time.LocalDateTime;
+
+public class Commands {
+
+    /**
+     * List of both primary and secondary commands.
+     */
+    public enum COMMANDS {BYE, LIST, TODO, DEADLINE, EVENT, MARK, UNMARK, DELETE, BY, FROM, TO, SORT, FIND, UNKNOWN}

```

Comment:
I like how you classified the commands using enums too. Good job for going the extra mile!